### ISSUE TITLE
[BUG] Dissapearing curios chest slot packpack

### ISSUE THREAD

**Issue by AikaBack** [Quarris/EnigmaticGraves#49]
**Describe the bug**
Sophisticated backpacks' backpack dissapears after dying and retrieving grave from EnigmaticGraves mod.

**To Reproduce**
1. Install curios, sophisticated backpacks and enigmatic graves latest versions on 1.19.2
2. Equip any backpack in chest slot
3. Die
4. Retrieve grave
5. See no backpack
6. 
**Expected behavior**
Backpack should be found on chest baubles after retrieving grave, or at least be in inventory.

**Versions**
 - Minecraft-Mod: 1.18.2-1.7.6
 - Forge: 43.2.7
 - sophisticatedbackpacks-1.19.2-3.18.40.779
 - curios-forge-1.19.2-5.1.3.0

**Additional  info**
There is no crash log.

**Comment by AikaBack** [comment]
minecraft-mod verssion is 1.19.2-1.7.6*

**Comment by orionSquared** [comment]
This is not just happening to backpacks, this is happening to anything in a curios/baubles slot. I'm playing on version 1.18 and I've literally lost all my baubles and my backpack that was full of stuff because my grave didn't keep that info. Idk if you have a fix for it but I've died so many times just trying to retrieve that grave that the game has no longer saved that grave for me to restore it. So...it's just gone.

**Comment by AikaBack** [comment]
yeah he said if you pay him he will fix this gamebreaking bug, but for now he doesnt care

**Comment by Quarris** [comment]
I'm getting the project up today and will try to replicate it tomorrow when I get up for 1.18+

**Comment by AikaBack** [comment]
this bug is on 1.19 as well. Focus on that first.

**Comment by orionSquared** [comment]
Yo, chill tf out. There's no need to be hostile or rude to the mod dev. They're doing the best they can for literally no benefit to themselves. Don't insinuate that they only will fix bugs for payment or demand they focus on certain things. Just report the bug and go live your life. Mod developing is a thankless job that thousands of us rely on to play the game we love and the last thing we need to do is bully those who are doing their best and delivering top-tier mods for us. 

Hell, I almost didn't even add to this bug report since it's already on their radar. Only reason I added to it was because it wasn't just backpacks, but other curio items as well. I added my piece and then moved on with my life. Please do the same.




Quarris, idk if this is helpful but it wasn't doing it when I first launched the server. It wasn't until about a week into playing that it started happening. Not sure if there's like a file size limit on the player data? Idk if that's helpful, but thought I'd add it. 

Also thank you for making such an amazing mod. Hands-down the best pure gravestone mod we have in minecraft. It's an auto-inclusion any time I play/build a modpack. <3

**Comment by AikaBack** [comment]
who are you to tell me to chill? Ive lost all my items using this mod, then rechecked and reconfirmed that this mod indeed destroys items, made bug report, got no response, then for 4 months ive been asking him to at least CHECK the bug, he brushed me off each time and then asked me MONEY to fix GAMEBREAKING BUG. 
Then you come and he instantly replies that he will check? Beyond rude

**Comment by Quarris** [comment]
Alright just drop it. I said I'll work on it.

Anyone has potential server logs when this happened? No worries if they're gone right now but they would be helpful

**Comment by AikaBack** [comment]
Here is latest log of our server, in which i again lost backpack and we never played again. It was 29th of may this year. 
[latest.log](https://github.com/Quarris/EnigmaticGraves/files/12433125/latest.log)

**Comment by Quarris** [comment]
Is this reproducable 100% of the time? I can't get it to work on 1.19.2 with just putting on a backpack.
Is it special backpack or literally any will work?
Does the bug still happen with just EG and Backpacks installed? Which pack was used?

**Comment by Quarris** [comment]
This is my setup and I cannot reproduce it with this. At least not by the steps describe initially
![image](https://github.com/Quarris/EnigmaticGraves/assets/6171764/05cdb553-b8a2-4a89-876a-c1119d976677)

**Comment by AikaBack** [comment]
My modpack had many slots for baubles, try installing enigmatic legacy

**Comment by AikaBack** [comment]
Ive found exactly when it happens, when some curio slot gives another curio slot and you die, grave deletes backpack.
Mod enigmatic legacy has item Dormant eye, when you right click it, it activates and you can equip it in amulet slot, and then eye gives you another amulet slot to equip your standard amulet from enigmatic legacy, after you equip both amulet slots and die, that is when backpack dissapears.

**Comment by AikaBack** [comment]
Here is video proof: https://drive.google.com/file/d/1ZEfQ0cGCeLvg4MJ5LmUaOkO__idAPgiA/view?usp=sharing
Mods are:
![image](https://github.com/Quarris/EnigmaticGraves/assets/42915767/100b52ce-55cb-45a0-a69d-7da8da9b1183)

**Comment by AikaBack** [comment]
are you looking into it?

**Comment by Quarris** [comment]
Im working on it on my off days. I said I don't have a lot of spare time.

**Comment by Quarris** [comment]
Managed to replicate it and fix it. Will release in v1.7.8

**Comment by Quarris** [comment]
Should be fixed in 1.7.8 for 1.19 and 1.18. Let me know if the problem still persists

**Comment by AikaBack** [comment]
I tested and it really is fixed! THANK YOU man! This mod was the only one equipping all curios in their slots and not putting them in inventory, but this issue really bugged me, now I'll play my ideal modpack again, appreciate it.

**Comment by AikaBack** [comment]
Got a crash says something enigmatic gravestones curios 
[crash-2023-08-31_23.06.29-server.txt](https://github.com/Quarris/EnigmaticGraves/files/12489138/crash-2023-08-31_23.06.29-server.txt)
[latest.log](https://github.com/Quarris/EnigmaticGraves/files/12489139/latest.log)

**Comment by AikaBack** [comment]
well, while I went back to retrieve grave after that crash, I had died, got another grave, got same crash again, then went into creative to fly fast to my first grave, and grave was not there, I was holding grave finder with coordinates in my hand and on those coordinates where I died falling onto stalagmite there is no grave whatsoever, flying in spectator mode also did not reveal anything in proximity. Now I lost ALL items, not just backpack. I cant even
[crash-2023-09-01_12.47.25-server.txt](https://github.com/Quarris/EnigmaticGraves/files/12495785/crash-2023-09-01_12.47.25-server.txt)
[latest.log](https://github.com/Quarris/EnigmaticGraves/files/12495756/latest.log)

**Comment by Quarris** [comment]
That's not good. I'm also having ISP issues on my end so it'll take me longer to fix it.
Would you be able to send me `<worldSave>/data/enigmaticgraves.dat` as well?

**Comment by Quarris** [comment]
I believe I fixed the issue however I cannot restore old graves. What you could do is restore yourself a grave that is slightly older using `/enigmatic_graves <Player> restore death_<id>_<time> true`. You'd need to know roughly when you last died that had most of your stuff but it might work.

Will be fixed in 1.7.9

**Comment by AikaBack** [comment]
Here's dat file: https://dropmefiles.com/KadGD. I can note that I died falling right on top of stalagmite block, maybe this somehow prevented grave from spawning or smth? You should probably check it?
I had many empty curio slots, mod artifacts adds many slots that are always visible. I had no items that increased count of curio slots.

**Comment by AikaBack** [comment]
latest beta 1.7.11 b1 update behaves very strangely, I used command to get that grave, it gave most items but no curios and there was error in chat, and then after I died again and retrieving my new grave, grave didnt dissapear and I could get items from it endlessly, it just gives items every click and doesnt dissapear. 
[2023-09-02-1.log](https://github.com/Quarris/EnigmaticGraves/files/12504021/2023-09-02-1.log)
Installed 1.7.10 version and graves behaved normally.

**Comment by Quarris** [comment]
Thank you, i'm not ignoring this issue but again dont have a lot of spare time working on this. Will try to get some work on this next week though

**Comment by AikaBack** [comment]
Hey man, so Im playing with version enigmaticgraves-1.19.2-1.7.10 and my items from mod enigmatic legacy dont get placed into curios slots, they are put into inventory, I remember mod didnt to that previously, what changed?
